### Validez Paralela o alternativa

Se administran dos instrumentos equivalentes, es decir no es el mismo instrumento. Los instrumentos son equivalentes en contenidos, preguntas, instrucciones, duración, etc. Se administran a un mismo grupo dentro de un periodo de tiempo corto. Se mide la confiabilidad a través de la correlación de ambos puntajes

In [1]:
import pandas as pd
from scipy.stats import pearsonr

# Lee el archivo Excel en un DataFrame de pandas
df = pd.read_excel('data/Resultados_Simulados_Formas_Paralelas.xlsx')

# Calcula la suma de los puntajes de las formas A y B
puntajes_totales_A = df.loc[:, 'Ítem_A_1':'Ítem_A_10'].sum(axis=1)
puntajes_totales_B = df.loc[:, 'Ítem_B_1':'Ítem_B_10'].sum(axis=1)

# Calcula la correlación de Pearson entre las sumas totales de las formas A y B
correlacion, p_valor = pearsonr(puntajes_totales_A, puntajes_totales_B)

# Imprime la correlación y el valor p
print(f'Correlación de Pearson: {correlacion}')
print(f'Valor p: {p_valor}')


Correlación de Pearson: 0.9567977517345541
Valor p: 2.236459386771317e-54


### Correlación de Pearson: 0.9567977517345541
La correlación de Pearson es una medida que indica el grado de relación lineal entre dos variables cuantitativas. El valor puede variar entre -1 y 1, donde:

1 indica una correlación positiva perfecta (a medida que una variable aumenta, la otra también lo hace en proporción directa).
-1 indica una correlación negativa perfecta (a medida que una variable aumenta, la otra disminuye en proporción directa).
0 indica que no hay correlación lineal entre las variables.
Un valor de 0.9567977517345541 es muy cercano a 1, lo que sugiere una fuerte correlación positiva entre tus dos conjuntos de datos. Esto significa que a medida que los valores de una variable aumentan, los valores de la otra variable tienden a aumentar en una proporción muy similar, indicando una relación lineal muy fuerte entre ellos.

### Valor p: 2.236459386771317e-54
El valor p es un indicador de la significancia estadística de los resultados obtenidos. En términos simples, te dice si el resultado observado es probablemente real o si ocurrió por casualidad.

Un valor p bajo (generalmente menor que 0.05) indica que es muy improbable que la correlación observada sea resultado del azar. En otras palabras, hay suficiente evidencia para decir que existe una verdadera correlación entre las variables.
Un valor p alto sugiere que la correlación observada podría ser resultado del azar, y no hay suficiente evidencia para afirmar que existe una relación real entre las variables.
El valor p que has proporcionado, 2.236459386771317e-54, es extremadamente bajo, prácticamente cero. Esto significa que la fuerte correlación observada es altamente significativa desde el punto de vista estadístico, y es muy improbable que se deba al azar. 

### En resumen, puedes estar muy seguro de que la relación lineal que encontraste entre las dos variables es real y estadísticamente significativa.